test/conformance/api/v1/util.go

```diff
@@ -340,3 +343,22 @@ func validateImageDigest(imageName string, imageDigest string) (bool, error) {
 
 	return ref.Context().String() == digest.Context().String(), nil
 }
+
+// WaitForScaleToZero will wait for the specified deployment to scale to 0 replicas.
+// Will wait up to 6 times the ScaleToZeroGracePeriod (1 minute) before failing.
+func WaitForScaleToZero(t pkgTest.TLegacy, deploymentName string, clients *test.Clients) error {
+	t.Helper()
+	t.Logf("Waiting for %q to scale to zero", deploymentName)
+	ScaleToZeroGracePeriod := 30 * time.Second
```

Comment:
This should be a const. Also this does not match the comment.
And `sclaeToZeroGracePeriod` or just `scaleToZeroTimeout`.

Reply:
thanks, @vagababov!